### DISCUSSION TITLE
Why `password_does_not_meet_requirements_message` is defined private

### DISCUSSION THREAD

**Discussion by igor-alexandrov** [jeremyevans/rodauth#329] [General]
I want to reuse `password_does_not_meet_requirements_message` in some custom forms using InternalRequest feature, but it is defined as private. https://github.com/jeremyevans/rodauth/blob/master/lib/rodauth/features/login_password_requirements_base.rb#L87

At the same time `invalid_previous_password_message` and some other password validation methods are defined as public. Can anybody please explain why?

Thank you.

**Comment by jeremyevans** [comment]
In general, the default is to have methods private unless they are deliberately designed to be used publicly.  I don't have a problem changing the visibility of that method to public, please send a pull request if you would like that (make sure all overridden methods use the same visbility).

**Reply by igor-alexandrov** [reply]
Got it. Thank you Jeremy!
